src/core/AgnLocusStream.c

```diff
@@ -412,7 +417,8 @@ static void locus_stream_extend(AgnLocusStream *stream, AgnLocus *locus)
     {
       agn_locus_set_range(locus, locusrange.start,
                           locusrange.end + stream->delta);
-      if(stream->endmode >= 0 && !stream->skip_iiLoci)
+      if(stream->endmode >= 0 && !stream->skip_iiLoci &&
+			locusrange.end + stream->delta + 1 <= seqrange.end)
```

Comment:
Looks like many of the changes to this file are cosmetic (line width). This is one of the few changes related to length accounting, right?